### PR TITLE
Ignite versions fixed

### PR DESCRIPTION
**Pull request by agura**
### What is this PR for?
This PR fixes Apache Ignite version mentions in interpreter documentation.

### What type of PR is it?
[Improvement]

### CODE REVIEW

File: docs/interpreter/ignite.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ After configuring Ignite interpreter, create your own notebook. Then you can bin
 
 ![Binding Interpreters](../assets/themes/zeppelin/img/docs-img/ignite-interpreter-binding.png)
 
-For more interpreter binding information see [here](http://zeppelin.apache.org/docs/manual/interpreters.html).
+For more interpreter binding information see [here](http://zeppelin.apache.org/docs/0.6.2/manual/interpreters.html).
```
Comment (Leemoonsoo) [inline docs/interpreter/ignite.md]: How about use relative path here? `../manual/dynamicform.html` would work.

File: docs/interpreter/ignite.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ You can use Zeppelin to retrieve distributed data from cache using Ignite SQL in
...
-2. Examples are shipped as a separate Maven project, so to start running you simply need to import provided <dest_dir>/apache-ignite-fabric-1.2.0-incubating-bin/pom.xml file into your favourite IDE, such as Eclipse.
+1. Ignite provides examples only with source or binary release. Download Ignite [source release](https://ignite.apache.org/download.html#sources) or [binary release](https://ignite.apache.org/download.html#binaries) whatever you want. But you must download Ignite as the same version of Zeppelin's. If it is not, you can't use scala code on Zeppelin. You can find ignite version in Zeppelin at the pom.xml which is placed under `path/to/your-Zeppelin/ignite/pom.xml` ( Of course, in Zeppelin source release ). Please check `ignite.version` .
```
Comment (AhyoungRyu) [inline docs/interpreter/ignite.md]: I've just created #1584 to provide supported Ignite version info in [Zeppelin's supported interpreters table](https://zeppelin.apache.org/supported_interpreters.html#supported-interpreters). So how about updating this sentence

"You can find ignite version in Zeppelin at the pom.xml which is placed under `path/to/your-Zeppelin/ignite/pom.xml` ( Of course, in Zeppelin source release ). Please check `ignite.version` ."

to

"The supported Ignite version is specified in [Supported Interpreter table](https://zeppelin.apache.org/supported_interpreters.html#ignite) for each Zeppelin release. If you're using Zeppelin master branch, please see `ignite.version` in `path/to/your-Zeppelin/ignite/pom.xml`. "

File: docs/interpreter/ignite.md (hunk 1 above)
Comment (AhyoungRyu) [inline docs/interpreter/ignite.md]: @agura I think it should be `../manual/interpreter.html` not `../manual/dynamicform.html`.

**Comment by AhyoungRyu** [comment]
@agura Thanks for your quick response. Except for `../manual/interpreters.html#what-is-interpreter-setting` link, LGTM.

**Comment by agura** [comment]
@AhyoungRyu I don't see the `../manual/interpreters.html#what-is-interpreter-setting` link. Where is it?

**Comment by AhyoungRyu** [comment]
@agura `../manual/interpreters.html#what-is-interpreter-setting` will be connected to [https://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/manual/interpreters.html#what-is-interpreter-setting](https://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/manual/interpreters.html#what-is-interpreter-setting).

**Comment by agura** [comment]
@AhyoungRyu Are there something to do in order to merge this PR?

File: docs/interpreter/ignite.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ After configuring Ignite interpreter, create your own notebook. Then you can bin
 
 ![Binding Interpreters](../assets/themes/zeppelin/img/docs-img/ignite-interpreter-binding.png)
 
-For more interpreter binding information see [here](http://zeppelin.apache.org/docs/manual/interpreters.html).
+For more interpreter binding information see [here](../manual/dynamicform.html).
```
Comment (AhyoungRyu) [inline docs/interpreter/ignite.md]: You need to change this link `../manual/dynamicform.html` to `../manual/interpreters.html#what-is-interpreter-setting` :)

**Comment by AhyoungRyu** [comment]
@agura Thanks for taking care of the Ignite documentation. The update is looks good to me. 
Merge into master if there are no more discussions.

**Pull request closed without merging**